apps/forgejo/overlays/nishir-tailnet/kustomization.yaml

```
@@ -14,19 +13,20 @@ labels:
+  - path: patch-tcproute.yaml
+  - path: patch-httproute.yaml
     target:
       name: forgejo
       version: v1
-      kind: NetworkPolicy
+      kind: HTTPRoute
 secretGenerator:
   - name: forgejo-config
     files:
       - app.ini=forgejo/app.enc.ini
 patches:
```

YAMLlint (1.37.1)

[error] duplication of key "patches" in mapping

(key-duplicates)